### ISSUE TITLE
Add support for v5 of google and google-beta providers

### ISSUE THREAD

**Issue by busser** [terraform-google-modules/terraform-google-lb-http#360]
### TL;DR

The `google` and `google-beta` providers recently got a new major version, v5.
The modules in this repo currently don't support it:

```terraform
google = {
  source  = "hashicorp/google"
  version = ">= 4.50, < 5.0"
}
google-beta = {
  source  = "hashicorp/google-beta"
  version = ">= 4.50, < 5.0"
}
```

This is stopping users from migrating to the new major version.

**Comment by huntedman** [comment]
There's a pending PR for it, but no action from maintainers.

https://github.com/terraform-google-modules/terraform-google-lb-http/pull/364

**Comment by huntedman** [comment]
> Really sad that the Google Terraform module is not well maintained 😢 I have to migrate out and will really think about using these modules in the futur

To be honest, it's pretty easy and straightforward to solve via a fork. The downside being you need to maintain the code.

But Looking at the actual code in the modules, I ended up not even using this module, and getting a better idea of how to solve it for my use case.

**Comment by imrannayer** [comment]
@huntedman #364 is not the correct PR for this. PR for this is #363 . It is failing test which is blocking the rollout.

**Comment by imrannayer** [comment]
Fixed in #374

**Comment by clintonb** [comment]
@imrannayer any idea when we can expect a new release?
